test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpBlockTest/WithUnexpectedTransitionsInAttributeValue_Throws.stree.txt

```diff
@@ -1,38 +1,54 @@
-        SyntaxKind.LeftBrace;[{];
-    Markup block - Gen<None> - 18 - (1:0,1)
-        Tag block - Gen<None> - 18 - (1:0,1)
-            Markup span - Gen<Markup> - [<span] - SpanEditHandler;Accepts:Any - (1:0,1) - Tokens:2
-                SyntaxKind.OpenAngle;[<];
-                SyntaxKind.Text;[span];
-            Markup block - Gen<Attr:foo, foo='@(6:0,6),'@(15:0,15)> - 10 - (6:0,6)
-                Markup span - Gen<None> - [ foo='] - SpanEditHandler;Accepts:Any - (6:0,6) - Tokens:4
-                    SyntaxKind.Whitespace;[ ];
-                    SyntaxKind.Text;[foo];
-                    SyntaxKind.Equals;[=];
-                    SyntaxKind.SingleQuote;['];
-                Markup block - Gen<DynAttr:@(12:0,12)> - 1 - (12:0,12)
-                    Expression block - Gen<Expr> - 1 - (12:0,12)
-                        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (12:0,12) - Tokens:1
-                            SyntaxKind.Transition;[@];
-                        Code span - Gen<Expr> - [] - ImplicitExpressionEditHandler;Accepts:NonWhitespace;ImplicitExpression[RTD];K14 - (13:0,13) - Tokens:1
-                            SyntaxKind.Unknown;[];
-                Markup block - Gen<DynAttr: @(13:0,13)> - 2 - (13:0,13)
-                    Markup span - Gen<None> - [ ] - SpanEditHandler;Accepts:Any - (13:0,13) - Tokens:1
-                        SyntaxKind.Whitespace;[ ];
-                    Expression block - Gen<Expr> - 1 - (14:0,14)
-                        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (14:0,14) - Tokens:1
-                            SyntaxKind.Transition;[@];
-                        Code span - Gen<Expr> - [] - ImplicitExpressionEditHandler;Accepts:NonWhitespace;ImplicitExpression[RTD];K14 - (15:0,15) - Tokens:1
-                            SyntaxKind.Unknown;[];
-                Markup span - Gen<None> - ['] - SpanEditHandler;Accepts:Any - (15:0,15) - Tokens:1
-                    SyntaxKind.SingleQuote;['];
-            Markup span - Gen<Markup> - [ />] - SpanEditHandler;Accepts:None - (16:0,16) - Tokens:3
-                SyntaxKind.Whitespace;[ ];
-                SyntaxKind.ForwardSlash;[/];
-                SyntaxKind.CloseAngle;[>];
-    Code span - Gen<Stmt> - [] - SpanEditHandler;Accepts:Any - (19:0,19) - Tokens:1
-        SyntaxKind.Unknown;[];
-    MetaCode span - Gen<None> - [}] - SpanEditHandler;Accepts:None - (19:0,19) - Tokens:1
-        SyntaxKind.RightBrace;[}];
+CSharpCodeBlock - [0..20)::20 - [{<span foo='@ @' />}]
+    CSharpStatement - [0..20)::20
+        CSharpTransition - [0..0)::0 - Gen<None> - SpanEditHandler;Accepts:None
+            Transition;[<Missing>];
+        CSharpStatementBody - [0..20)::20
+            RazorMetaCode - [0..1)::1 - Gen<None> - SpanEditHandler;Accepts:None
+                LeftBrace;[{];
+            CSharpCodeBlock - [1..19)::18
+                MarkupBlock - [1..19)::18
```

Comment:
Hmm, shouldn't there be a C# marker before this?  If that's how things used to work that's fine, just seems odd

Reply:
Looks like this is how it used to work. There is a CSharp Marker at the end of this MarkupBlock but not before.